docs/community/contribution guidelines.rst

```diff
@@ -128,6 +128,118 @@ Create a pull request (from now on we will shorten it often to just *PR*) by vis
+|   }                                   |   }                                      |
+|                                       |                                          |
++---------------------------------------+------------------------------------------+
+
+Operator spacings
+-----------------
+
++---------------------------------------+------------------------------------------+
+| Correct                               | Incorrect                                |
++=======================================+==========================================+
+|                                       |                                          |
+|.. code-block:: c++                    |.. code-block:: c++                       |
+|   :emphasize-lines: 6                 |   :linenos:                              |
+|   :linenos:                           |                                          |
+|                                       |                                          |
+|   // Unary                            |   // Unary                               |
+|   i++;                                |   i ++;                                  |
+|   if (!b) {}                          |   if (! b) {}                            |
+|                                       |                                          |
+|   // Assignment, binary, ternary      |   // Assignment, binary, ternary         |
+|   y = m*x + b;                        |   y = m*x+b;                             |
+|   c = a | b;                          |   c=a|b;                                 |
+|   return condition ? true : false;    |   return condition?true:false;           |
+|                                       |                                          |
++---------------------------------------+------------------------------------------+
+| | To emphasize order no spacing is OK |                                          |
+|   sometimes                           |                                          |
+| | (line no 6)                         |                                          |
++---------------------------------------+------------------------------------------+
+
+Braces & more spaces
+--------------------
+
+.. code-block:: c++
+
+   /* All C++ reserved keywords where braces are inevitable
+    * Are placed on the same line
+    *
+    * Also make sure to leave one space before function brackets
+    */
+
+   if (condition) {
+       ...
+   } else {
+       ...
+   }
+
+   switch (value) {
+       case 1: ...
+       case n: ...
+   }
+
+   enum MyEnum {
+       ...
+   };
+   
+   // Same apply for C++ loops too
+
+   while (condition) { ... }
+
+   do {
+       ...
+   } while (condition);
+
+   for (const auto& item : list) {
+       ...
+   }
+
+   // User defined functions are exceptions
+   // - Brace on the new line
+   // - And no space before function bracket
+    
+   void my_function()
```

Comment:
Was defined we should keep this style below?
```c++
void
my_function()
{
}
```

Reply:
LOL the vote was 50-50 I didn't know what to choose without offending the other parties XD

Synfig and GTKMM code specifically tend to have lengthy return types so I voted your above style in the forum. Also, GTK dev team uses/recommends that in their codebase too.

I'll update the doc anyway... 😓